src/components/designer/link/TaroBackend.tsx

```diff
@@ -10,8 +10,8 @@ interface Props {
   to: LndNode;
 }
 
-const TaroLnd: React.FC<Props> = ({ from, to }) => {
-  const { l } = usePrefixedTranslation('cmps.designer.link.lndbackend');
+const TaroBackend: React.FC<Props> = ({ from, to }) => {
+  const { l } = usePrefixedTranslation('cmps.designer.link.tarobackend');
```

Comment:
The key should match the name of the component, case-sensitive.

Reply:
Done